app/views/add_menu.py

```diff
@@ -7,11 +7,13 @@
 class AddMenu(Resource):
     def post(self):
         user_request = request.get_json()
+        print(user_request)
```

Comment:
I see print statements here

Reply:
Alright am going to work on that

Reply:
I am planning to run tests transnight 
am sure I will come back tomorrow with 90% coverage